src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -82,6 +84,7 @@ const eventBufferLength: ReadonlyMap<string, number> = new Map([
 ]);
 
 export class EventManager implements IEventManager {
+  readonly NETWORK_DOMAIN_PREFIX = 'network.';
```

Comment:
Is this meant to be an immutable constant for every instance?

If yes, please add `static` here as well

https://google.github.io/styleguide/tsguide.html#identifiers-constants

(Another possibility is to move it outside the class; your call.)

Reply:
done